Add `paulburgess1357/nvim-mcp` (#2284)
rockerBOO/awesome-neovim: README.md

- [DrKJeff16/nvim-plugin-boilerplate](https://github.com/DrKJeff16/nvim-plugin-boilerplate) - A documented template for new plugins generated by a script. Includes tests, CI utilities, etc.
- [chrisgrieser/nvim-pseudometa-plugin-template](https://github.com/chrisgrieser/nvim-pseudometa-plugin-template) - Template for new Neovim plugins.

### Remote Control

> - [paulburgess1357/nvim-mcp](https://github.com/paulburgess1357/nvim-mcp) - MCP server that gives AI agents access to buffers, commands, and LSP diagnostics through Neovim's msgpack-RPC socket. Be sure to work safely when giving LLMs access to Neovim.

Reviewer: ```suggestion
- [paulburgess1357/nvim-mcp](https://github.com/paulburgess1357/nvim-mcp) - MCP server that grants access to buffers, commands, and LSP diagnostics through AI agents through Neovim's msgpack-RPC socket.
```

Author: Pushed a slight variant of this description and removed the warning as the readme now has a link at the top to the warning
